Tools/Taxonomy/Common/Options.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using CommandLine;
+
+namespace Taxonomy
+{
+    interface IOptions
+    {
+        [Value(0, MetaName = "source file path", HelpText = "Source file path to process.", Required = true)]
+        string SourceFilePath { get; set; }
+
+        [Value(1, MetaName = "target file path", HelpText = "Target file path to save.", Required = true)]
+        string TargetFilePath { get; set; }
+    }
+
+    [Verb("generatecwe", false, HelpText = "Generate CWE Sarif file")]
```

Comment:
>generatecwe [](http://example.com/codeflow?start=11&length=11)

change to generate-cwe and update the others as well

Reply:
good idea